src/components/tab-title/tab-title.scss

```diff
@@ -7,7 +7,40 @@
 :host([layout="center"]) {
   @apply my-0 mx-5 text-center;
   flex-basis: theme("spacing.48");
-  margin: auto;
+  .content {
+    @apply m-auto;
+  }
+}
+
+// center the text visually and not affected by the x button so as to avoid moving when on or off
+:host([layout="center"][closable]) {
+  .content {
+    padding-inline-start: 32px; //28px button width + 0.25rem padding
+  }
+}
+
+:host([layout="center"][bordered][closable][scale="s"]) {
+  .content {
+    padding-inline-start: 36px; //28px button width + 0.5rem padding
+  }
+}
+
```

Comment:
I worked the offset px/rem amount for the text to stay static in the center when buttons are added or removed, but I'm not sure how it needs to be expressed in terms of tokens. 

@alisonailea suggested I use format var(--core-sizing-9) to reference button width, but the offset here is cumulative of the width and the existing padding. 

Could I get some help with this?  @driskull